packages/apps/docs/package.json

```diff
@@ -100,6 +103,7 @@
     "storybook": "^7.0.20",
     "storybook-addon-next-router": "~4.0.2",
     "storybook-dark-mode": "^3.0.0",
-    "typescript": "5.0.4"
+    "typescript": "5.0.4",
+    "dotenv": "~16.0.3"
```

Comment:
Please run `rush format`

Reply:
I thought that would go automatically.
not anymore?

Reply:
`rush add` never does/did this properly